### ISSUE TITLE
8$: Suggest 'await' for 'async foo.bar()'

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#770]
real-world code:
```js
    expect(async children[""].getContentsAsync()).toEqual(new Buffer("HTML goes here"));
```
`async` should have been `await`. qljs reported "E0176: missing arrow operator for arrow function", which is unhelpful.
